gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-environments.yaml

```diff
@@ -1386,6 +1386,23 @@ components:
                             links:
                                 $ref: "#/components/schemas/Links"
 
+        ClustersResponse:
```

Comment:
Can you complete the open API paths so that the response can be used by the search to be defined?